src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.AssemblyResolution.targets

```diff
@@ -110,6 +114,7 @@ _ResolveAssemblies MSBuild target.
       <_ResolvedAssemblies          Include="@(ResolvedAssemblies)" />
       <_ResolvedUserAssemblies      Include="@(ResolvedUserAssemblies)" />
       <_ResolvedFrameworkAssemblies Include="@(ResolvedFrameworkAssemblies)" />
+      <_ResolvedSymbols             Include="@(ResolvedSymbols)" />
```

Comment:
Comparing with the Legacy `_PrepareAssemblies` we only set the `@(_ResolvedSymbols)` ItemGroup when `AndroidLinkMode ` == `None` [1]. We should probably do the same for .net 5?


[1] https://github.com/xamarin/xamarin-android/blob/master/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets#L376

Reply:
Then again, that might break things if the Task always doing the work... 🤔

Reply:
I think it just does it two different ways:

https://github.com/xamarin/xamarin-android/blob/0907f09f9c2073522adb66d717c2726be3712208/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets#L358-L362

https://github.com/xamarin/xamarin-android/blob/0907f09f9c2073522adb66d717c2726be3712208/src/Xamarin.Android.Build.Tasks/Xamarin.Android.Legacy.targets#L378

We don't need to check `File.Exists()` because the `@(ResolvedFileToPublish)` from the dotnet/sdk reliably tells us which files we need.

Reply:
ok col